pyproject.toml

```diff
@@ -6,7 +6,6 @@ authors = ["DesignSafe-CI <designsafe-ci@tacc.utexas.edu>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-agavepy = "1.0.0a12"
 pytas = {git = "https://bitbucket.org/taccaci/pytas.git", tag = "v1.7.0"}
 attrdict = { git = "https://github.com/DesignSafe-CI/AttrDict", rev = "83b779ee82d5b0e33be695d398162b8f2430ff33" }
```

Comment:
```suggestion
```
this was to workaround issues with agavepy so could be dropped with agavepy

Reply:
Thanks, removed